### PR TITLE
Adding Kodi, Octave Online, Pterodactyl 

### PR DESCRIPTION
**Pull request by d1nuc0m**
- One item per committ
- Closes issues #2130, #2931 
- Updates issue #1555

Default text below for readability:
<details>

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.
<\details>

### CODE REVIEW

**Comment by d1nuc0m** [comment]
@nodiscc I've seen the label "feedback needed", which additional information is needed? :)

File: README.md [hunk 1]
```diff
@@ -952,13 +952,15 @@ Games, game servers and control panels.
 - [Hextris](https://github.com/Hextris/hextris) - Fast paced HTML5 puzzle game inspired by Tetris. ([Demo](https://hextris.github.io/hextris)) `GPL-3.0` `HTML5`
 - [Legend of the Green Dragon](https://github.com/lotgd/core) - Legend of the Green Dragon is a text-based RPG originally developed by Eric Stevens and JT Traub as a remake of and homage to the classic BBS Door game, Legend of the Red Dragon, by Seth Able Robinson. ([Demo](http://lotgd.net/)) `AGPL-3.0` `PHP`
 - [Lila](https://lichess.org/) - The forever free, adless and open source chess server powering lichess.org, with official iOS and Android client apps. ([Source Code](https://github.com/ornicar/lila)) `AGPL-3.0` `Scala`
+- [LinuxGSM](https://linuxgsm.com/) - CLI tool for deployment and management of dedicated game servers on Linux: more than 120 games are supported. ([Source Code](https://github.com/GameServerManagers/LinuxGSM)) `MIT` `Shell`
```
Comment (nodiscc) [inline README.md:955]: Same problem as https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2857, I'm not sure we should include it (it's a command-line administration tool). Please remove it and open a separate issue if needed.

File: README.md [hunk 2]
```diff
@@ -1154,6 +1156,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Icecast 2](https://icecast.org) - streaming audio/video server which can be used to create an Internet radio station or a privately running jukebox and many things in between. ([Source Code](https://gitlab.xiph.org/xiph/icecast-server), [Clients](https://icecast.org/apps/)) `GPL-2.0` `C`
 - [Jellyfin](https://jellyfin.org) - Media server for audio, video, books, comics, and photos with a sleek interface and robust transcoding capabilities. Almost all modern platforms have clients, including Roku, Android TV, iOS, and Kodi. ([Demo](https://demo.jellyfin.org/stable), [Source Code](https://github.com/jellyfin/jellyfin)) `GPL-2.0` `C#`
 - [Karaoke Forever](https://www.karaoke-forever.com) - Host awesome karaoke parties where everyone can easily find and queue songs from their phone's web browser. The player is also browser-based with support for MP3+G, MP4 and WebGL visualizations. ([Source Code](https://www.karaoke-forever.com/repo)) `ISC` `Nodejs`
+- [Kodi](https://kodi.tv/) - Multimedia/Entertainment center, formerly known as XBMC. Runs on Android, BSD, Linux, macOS, iOS and Windows. ([Source Code](https://github.com/xbmc/xbmc)) `GPL-2.0` `C++`
```
Comment (nodiscc) [inline README.md:1159]: Kodi is great, but I'm not sure it counts as self-hosted? As far as I know it doesn't have a remotely accessible interface, it's a purely local media player. If I'm correct please remove it from this PR.

**Review by nodiscc — CHANGES_REQUESTED** [review]
@d1nuc0m Please address the requested changes/questions, copy back the [pull request template](https://raw.githubusercontent.com/awesome-selfhosted/awesome-selfhosted/master/.github/PULL_REQUEST_TEMPLATE.md) to the pull request description, and check the relevant boxes.

File: README.md (hunk 2 above)
Comment (d1nuc0m) [reply README.md:1159]: Kodi has [a web interface](https://kodi.wiki/view/Web_interface) that can be configured and used to stream content to other devices. See also [[1]](https://www.makeuseof.com/tag/how-to-use-kodi-web-interface/) [[2]](https://www.addictivetips.com/media-streaming/kodi/web-interface-access-files/). Is it enough to count it as self-hosted?

File: README.md (hunk 1 above)
Comment (d1nuc0m) [reply README.md]: Removed it

File: README.md (hunk 2 above)
Comment (d1nuc0m) [reply README.md:1158]: Ping @nodiscc

Comment (nodiscc) [reply README.md:1158]: > Kodi has [a web interface](https://kodi.wiki/view/Web_interface) that can be configured and used to stream content to other devices

Ok :+1:

**Review by nodiscc — APPROVED** [review]
Thanks

**Comment by d1nuc0m** [comment]
> Thanks

You're welcome

**Pull request merged**